singlefactorauth/src/main/java/com/web3auth/singlefactorauth/SingleFactorAuth.kt

```diff
@@ -122,13 +121,16 @@ class SingleFactorAuth(singleFactorAuthArgs: SingleFactorAuthArgs) {
 
     private fun getRetrieveSharesResponse(
         loginParams: LoginParams,
-        details: NodeDetails,
+        nodeDetails: NodeDetails,
         pubDetails: TorusPublicKey
-    ): RetrieveSharesResponse {
-        if (pubDetails.typeOfUser.equals(TypeOfUser.v1)) {
-            torusUtils.getOrSetNonce(pubDetails.x, pubDetails.y, false).get()
+    ): TorusKey {
+        if (pubDetails.metadata.typeOfUser.equals(TypeOfUser.v1)) {
+            MetadataUtils.getOrSetNonce(
+                "", pubDetails.metadata.pubNonce.x, pubDetails.metadata.pubNonce.y,
+                0, null, false, null
```

Comment:
We should not be calling MetadataUtils directly outside of TorusUtils.

Please see here for how this should be implemented:
https://github.com/Web3Auth/single-factor-auth-swift/blob/main/Sources/SingleFactorAuth/SingleFactorAuth.swift#L30-L77